src/foam/nanos/notification/services.jrl

```diff
@@ -78,7 +78,13 @@ p({
   "client": """
     {
       "of": "foam.nanos.notification.Notification",
-      "cache": false,
+      "cache": true,
+      "pollingInterval": 25000,
+      "pollingProperty": {
+        "class":"__Property__",
+        "forClass_":"foam.nanos.notification.Notification",
+        "name":"created"
+      },
```

Comment:
This should be done by the service worker, not using polling, we removed this recently